strings/gimmick_objectives.txt

```diff
@@ -0,0 +1,31 @@
+Convince the crew there is another antagonistic force acting on the station besides covert Syndicate operatives.
+Whenever an operative is arrested, break them free. The Syndicate looks out for its own.
+Whenever an operative is arrested, kill them. The Syndicate does not tolerate failure.
+Promote hostilities and tensions between different alien species. Pit those filthy xenos against each other!
+Start an underground cagefighting club. Kidnap or recruit participants.
+Single out a single species and make the rest of the crew hate them as much as possible.
+We've been taking notes from the British. Get as many people on the station addicted to drugs as possible!
```

Comment:
the british reference here isnt necessary

Reply:
removed